dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -48,4 +49,4 @@
   .field.teacher-only-markdown
 
     %textarea#level_teacher_markdown_textarea
-    #level_teacher_markdown_preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
is there a reason this isn't all in the _teacher_only_markdown partial?

Also, what's the difference between how we use this `#level_teacher_markdown_textarea_preview` div and how we use the `#level_teacher_markdown_preview` div?

Reply:
The partial only works on non-DSL defined levels because we store the teacher only markdown differently on DSL and non-DSL levels. We need to use `initializeEmbeddedMarkdownEditor` instead of `initializeCodeMirror` in order to get the right output in the level file. This also is why we need to use `#level_teacher_markdown_textarea_preview on DSL because thats what works with `initializeEmbeddedMarkdownEditor` vs #level_teacher_markdown_preview which works with `initializeCodeMirror`. All of this to say when we move away from DSL levels we should be able to bring these to together but for right now this was the quickest fix.

Reply:
Ah, gotcha. I'd love to see some comments in the relevant files explaining this distinction, but otherwise this LGTM